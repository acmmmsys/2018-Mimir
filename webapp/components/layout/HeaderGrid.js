import React, { Component } from 'react';
import { Grid } from 'semantic-ui-react'

import 'style/layout/GridLayout.scss';

class HeaderGrid extends Component {

    static defaultProps = {
        children: []
    }

    render() {
        return (
            <Grid 
                columns={this.props.children.length}
                className='grid-header' 
                verticalAlign='middle'
            >
                {this.props.children}
            </Grid>
        )
    }
}

export default HeaderGrid;
