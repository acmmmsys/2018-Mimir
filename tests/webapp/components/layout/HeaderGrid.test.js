import React from 'react';
import HeaderGrid from '../../../../webapp/components/layout/HeaderGrid'
import renderer from 'react-test-renderer';

let wrapper;

test('HeaderGrid does render', () => {
    wrapper = shallow(
        <HeaderGrid>

        </HeaderGrid>
    );
    expect(wrapper).toMatchSnapshot();
});
