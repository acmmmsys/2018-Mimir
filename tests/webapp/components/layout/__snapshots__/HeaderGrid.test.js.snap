// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderGrid does render 1`] = `
<Grid
  className="grid-header"
  columns={0}
  verticalAlign="middle"
/>
`;
